MapboxGeocoder/MBPlacemark.swift

```diff
@@ -486,6 +491,8 @@ open class GeocodedPlacemark: Placemark {
         return properties?.maki
     }
     
+    open var relevance: Double?
```

Comment:
Should we expose this property to Obj-C and workaround bridgability with a -1 constant?

Reply:
Yes, but I’m uncertain what the magic number should be – whether it should be –1 or whether it should be `Double.max` to avoid a collision with a legitimate value.